src/java/org/apache/cassandra/schema/ColumnMetadata.java

```diff
@@ -115,6 +118,12 @@ public boolean isPrimaryKeyKind()
     @Nullable
     private final ColumnMask mask;
 
+    @Nonnull
+    private ColumnConstraints columnConstraints;
+
+    @Nonnull
```

Comment:
Don't need @Nonnull on a primitive do you? It's nonnull by definition I thought.